src/pages/editor-page/canvas/toolbar/toolbar.tsx

```diff
@@ -44,29 +44,57 @@ export const Toolbar: React.FC<ToolbarProps> = () => {
     };
 
     return (
-        <div className="px-1">
-            <Card className="shadow-none p-0 bg-secondary h-[44px]">
-                <CardContent className="p-1 flex flex-row h-full items-center">
-                    <ToolbarButton onClick={updateDiagramUpdatedAt}>
+        <div className="px-1 md:mb-0 mb-20">
```

Comment:
When added the "top-center" position property to the controls in the canvas.tsx component, the changes in this file can be reverted as the whole toolbar is aligned in the top center of the canvas already.